README.md

```diff
@@ -1,2 +1,40 @@
-# zlc
-Zowe Leadership Committee collaboration
+# Overview
+
+This repository contains the materials of how the Zowe project operates, along with meeting notes and materials from regularly scheduled meetings of the Zowe Leadership Committee (ZLC).
+
+# Zowe Leadership Committee (ZLC)
+
+The Zowe Leadership Committee (ZLC) oversees all code development and release management for the Zowe project. In particular, the ZLC has the following roles:
+
+* Receive proposals for sub-projects leveraging Zowe technology and will decide, via a vote, on whether the sub-project is accepted.
+*	Be responsible for working with the OMP Marketing Committee to ensure messaging and content is consistent with the vision and mission of Zowe.
+*	Create and maintain development iteration criteria and release train to drive framework development and deployment
+*	When required, vote on revoking commit rights
+
+The ZLC is composed of six (6) members of the Zowe community. Initially IBM, CA Technologies, and Rocket Software have each nominated two indivuduals to serve on the ZLC while the Zowe community builds committer diversity, for a term that will end by July 31, 2019.
+
+To qualify for nomination to serve on the ZLC, the individual must be a committer on a Zowe sub-project. No more than 2 individuals on the ZLC can be afflilated with the same company.
+
+## Current ZLC voting members
+
+All members of the Zowe community are welcome to participate in meetings and discussion on the Zowe project. For matters requiring a vote of the ZLC ( such as approving a new sub-project or approving a Zowe release ), the following individuals are considered voting members of the ZLC:
+
+- Matt Hogstrom, Chairperson
+- Mark Ackert
+- Jean-Philippe Linardon
+- Sean Grady
+- Bruce Armstrong
+- Jean-Louis Vignaud
+
+## ZLC meetings
+
+All ZLC meeting are posted on the [Zowe Development Calendar](https://lists.openmainframeproject.org/g/zowe-dev/calendar). You can [view past meeting notes and materials](meetings).
+
+# Processes and procedures
+
+The Zowe project is self governed by the Zowe community, with each sub-project setting thier own policies for determining . The ZLC has estalished a number of governance processes as outlined below:
+
+- [Contribution Guidelines](https://zowe.org/contribute/)
+- [Project Role Definitions](process/roles.md)
+- [Code of Conduct](https://www.linuxfoundation.org/code-of-conduct/) - Zowe community members are bound to the Linux Foundation Code of Conduct.
```

Comment:
The very same CoC that is causing drama and threats within the Linux community? Let's avoid this until the Linux community at large settles in one direction or another. I don't want Zowe to get caught in the middle of controversy.

Reply:
Entirely different one - we use this for LF events and has existed long before the kernel community has looked into establishing one.

Reply:
This looks like the CoC referenced is for conferences.  I don't see an issue with it.  Honestly, the code of conduct in its current form focus a LOT on sexual issues and I think tries to hard but the intent seems fine to me.  I think we let it go.

Reply:
Code of conducts are important to get right because they establish a culture that surrounds a project. As you said, that one seems more suited to conferences than code. Here's an alternative CoC which I found to be a good fit, from KDE: https://www.kde.org/code-of-conduct/

Reply:
If this is a large concern, I'd suggest raising to the Open Mainframe Project TSC.

Reply:
@1000TurquoisePogs Are you OK with the current COC reference? I'd like to merge this PR to master and close out the waffle ticket.